static/js/pivotal.js

```diff
@@ -176,18 +232,18 @@
         (function(project) {
           var url = diffs[project];
           // hashes: currentCommit...latestCommit
-          var hashes = (url.substr(url.lastIndexOf('/') + 1)).split('...');
-          var proj_name = url.match(config.github.url_compare_regexp)[1];
-
-          getGithubPivotalIDs(GITHUB_TOKEN, proj_name, hashes[0], hashes[1], function(commits) {
-            var messages = [];
-            for (commit in commits) {
-              messages.push(commits[commit]['commit']['message']); // get all github commit messages
-            }
+          var commitHashes = (url.substr(url.lastIndexOf('/') + 1)).split('...');
+          var repositoryName = url.match(config.github.url_compare_regexp)[1];
 
-            var pivotal_ids = getCommitIDs(messages); // extract pivotal ticket IDs
+          getGithubCommits(GITHUB_TOKEN, repositoryName, commitHashes[0], commitHashes[1], function(commits) {
```

Comment:
Since both hashes are conceptually close, you can maybe leave them as a pair (the array) and pass it `getGithubCommits`. I'm saying that because usually when you start having a lot of parameters to a function (read more than 3), it's good to group them in higher level objets.